src/status_im/navigation/screens.cljs

```diff
@@ -338,15 +337,6 @@
                  :modalPresentationStyle :overCurrentContext}
      :component enable-notifications/view}
 
-    {:name      :screen/onboarding.identifiers
```

Comment:
I think we can also keep the screen definition here. 
Imo we only need to update the navigation